jsgui/src/main/webapp/index.html

```diff
@@ -18,6 +18,9 @@
 
     <!-- Clocker Branding. -->
     <link rel="stylesheet" href="/assets/css/clocker.css">
```

Comment:
`brooklyn.css` contains some changes for codemirror and misc, believe they need to be moved over.